frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -130,10 +139,35 @@ public void handle(long l) {
 				}
 			}
 		};
-
 		timer.start();
 	}
 
+	public void throwFrisbee() {
+		Timeline timelineFrisbee = new Timeline();
+        timelineFrisbee.setCycleCount(1);
+        KeyValue xKV = new KeyValue(this.frisbeeXPosition, 1000);
+        KeyValue yKV = new KeyValue(this.frisbeeYPosition, 100, new Interpolator() {
+            @Override
+            protected double curve(double t) {
```

Comment:
Ich verstehe die Funktion nur halb, aber ist die Berechnung ausreichend, um den Aufprallwinkel des Frisbees und ein zurückwerfen zu berechnen, wenn der andere Character den Frisbee fängt? Oder gehen wir jetzt den Weg des "manuellen" werfens?

Reply:
1. Generell zur Berechnung: Man tippt oder klickt auf die Frisbee und dann werden sowohl die beiden endValues für xKV und yKV in kleinem Ausmaß zufällig variiert, so dass der Fänger sich bewegen muss, damit die Frisbee nicht auf den Boden fällt.
2. Zum Ablauf: Wenn der Mitspieler die Frisbee gefangen hat, geht sein Arm leicht nach unten. Tippt man dann auf die Frisbee, geht der Arm wieder nach oben und die Frisbee fliegt mit variierter Distanz und Höhe und in Richtung Mitspieler.
3. Premium-Lösung: Man zieht die Frisbee und per Drag wird der Spieler hierdurch leicht gedreht (quasi wie bei einem Katapult). Per X-Auslenkung steuert man damit dann den Einfluss auf die Geschwindigkeit der Frisbee (erster Parameter der Keyframes xKF bzw. yKF). Vielleicht auch ganz spaßig … ; )